.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,5 +3,29 @@
 _Please add a description of the changes that this PR introduces and the files that
 are the most critical to review._ 
 
-Closes: #XXX
+If this PR is non-trivial/large/complex, please ensure that you have either
+created an issue that the team's had a chance to respond to, or had some
+discussion with the team prior to submitting substantial pull requests. The team
+can be reached via GitHub Discussions or the Cosmos Network Discord server in
+the #cometbft channel. GitHub Discussions is preferred over Discord as it
+allows us to keep track of conversations topically.
+https://github.com/cometbft/cometbft/discussions
+
+If the work in this PR is not aligned with the team's current priorities, please
+be advised that it may take some time before it is merged - especially if it has
+not yet been discussed with the team.
+
+See the project board for the team's current priorities:
+https://github.com/orgs/cometbft/projects/1
+
+-->
+
+---
+
+#### PR checklist
+
+- [ ] Tests written/updated
+- [ ] Changelog entry added in `.changelog` (we use
```

Comment:
same with other comment, usually we just use github's generate feature when publishing a new release in all of the other repos, but if we want to start using this I'm not against it